content/Hardware Support/GIGA R1/Configure-GIGA-R1-WiFi-Portenta-H7-and-Portenta-Machine-Control-for-Over-The-Air-OTA-uploads.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Configure GIGA R1 WiFi, Portenta H7, and Portenta Machine Control for Over-The-Air (OTA) uploads
+id: 12370721200540
+---
+
+Learn how to enable [Over-The-Air (OTA) uploads](https://docs.arduino.cc/arduino-cloud/features/ota-getting-started/) for:
+
+* GIGA R1 WiFi
+* Portenta H7
+* Portenta Machine Control
+
+Follow these steps if you experience any of these errors when attempting an OTA upload:
+
+* `Error while mounting the filesystem. Err = -5`
+* `Arduino_Portenta_OTA::begin() failed with error code -3`
+
+---
+
+## 1. Connect your board to Arduino IDE
+
+> [!IMPORTANT]
+> The required sketch examples are currently not available in the Web Editor.
+
```

Comment:
```suggestion
```

Turns out this information is outdated.

I'll create a new task to update this article with instructions for Cloud Editor.